tests/cypress/util.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Function that performs a programmatic login
+ * @param {Cypress} cy
+ */
+export function login(cy, cb) {
```

Comment:
Perform a login without having to explicitly fill in the login form (as a util in all tests that don't test the login page). It is still necessary to visit the login page to get a session id as well as the CSRF/ViewState (as far as I found out).